### PR TITLE
Allow properties when exporting SecurityAnalysisResult to XML

### PR DESCRIPTION
**Pull request by tadam50**
Signed-off-by: Thomas ADAM <tadam@silicom.fr>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
No

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Feature

**What is the current behavior?** *(You can also link to an open issue here)*
SecurityAnalysisResult is not export into XML format 

**What is the new behavior (if this is a feature change)?**
SecurityAnalysisResult can be exported into XML format 

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

**Review by MioRtia — COMMENTED** [review]
Not sure a cne parent module is needed, you can directly put the cne-converter module. Otherwise, the code is going great!
Don't forget to eventually move this PR to [powsybl/powsybl-entsoe](https://github.com/powsybl/powsybl-entsoe)

File: cne/cne-converter/src/main/java/com/powsybl/cne/converter/model/MonitoredRegisteredResource.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    private final String equipmentName;
+
+    private final List<Measurement> measurementList = new ArrayList<>();
+
+    public MonitoredRegisteredResource(Collection<LimitViolation> limitViolations) {
+        Objects.requireNonNull(limitViolations);
+        if (limitViolations.isEmpty()) {
+            throw new IllegalArgumentException("LimitViolation list cannot be empty");
+        }
+        LimitViolation first = limitViolations.stream().findFirst().get();
```
Comment (MioRtia) [inline cne/cne-converter/src/main/java/com/powsybl/cne/converter/model/MonitoredRegisteredResource.java:29]: You can reduce this to:
```java
LimitViolation first = limitViolations.stream().findFirst().orElseThrow(() -> new IllegalArgumentException("..."));
 ```

**Comment by tadam50** [comment]
> Not sure a cne parent module is needed, you can directly put the cne-converter module. Otherwise, the code is going great!
> Don't forget to eventually move this PR to [powsybl/powsybl-entsoe](https://github.com/powsybl/powsybl-entsoe)

I kept this PR in order to add new export method into SecurityAnalysisResultExporter & SecurityAnalysisResultExporters

**Review by MioRtia — APPROVED** [review]

**Comment by MioRtia** [comment]
@tadam50 It is okay for me but could you please add unit tests for this PR?

**Comment by MioRtia** [comment]
I think you can add a default implementation:
```java
default void export(SecurityAnalysisResult result, Writer writer) {
        export(result, null, writer);
    }
```

**Review by MioRtia — APPROVED** [review]

**Pull request merged**